distutils/cygwinccompiler.py

```diff
@@ -276,9 +288,12 @@ def __init__(self, verbose=False, dry_run=False, force=False):
         self.set_executables(
             compiler=f'{self.cc} -O -Wall',
             compiler_so=f'{self.cc} -shared -O -Wall',
+            compiler_so_cxx=f'{self.cxx} -mdll -O -Wall',
```

Comment:
In https://github.com/pypa/distutils/pull/184, the above line was changed from `-mdll` to `-shared`. Does the same change need to apply here?

Reply:
Most likely want to keep those with the same parameters.

Reply:
Done in 95088c588.